iroha_p2p/src/poll.rs

```diff
@@ -0,0 +1,204 @@
+use std::collections::HashMap;
```

Comment:
I guess this module can be removed then. Right?

Reply:
Yep.